.babelrc.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
       ],
     },
     'web-development': {
-      presets: [['@babel/preset-env', { modules: false }], '@babel/preset-react'],
+      presets: ['@babel/preset-env', '@babel/preset-react'],
```

Comment:
there should be a new environment `web-test` in that you should make these changes and keep rest of them untouched

Reply:
Why do we need different environment for tests? shouldn't tests run on the same environment as production/development?

Reply:
In fact, shouldn't we run our tests scripts in production environment like this:

**From**
```
    "test:native": "BABEL_ENV=development jest -c ./jest.native.config.js",
    "test:native:coverage": "yarn run test:native --coverage",
    "test:web": "BABEL_ENV=web-development jest -c ./jest.web.config.js",
    "test:web:coverage": "yarn run test:web --coverage"
```

**To**
```
    "test:native": "BABEL_ENV=production jest -c ./jest.native.config.js",
    "test:native:coverage": "yarn run test:native --coverage",
    "test:web": "BABEL_ENV=web-production jest -c ./jest.web.config.js",
    "test:web:coverage": "yarn run test:web --coverage"
```

Reply:
A unit test would be environment agnostic, we will need a different babel config. We will need the environment as production for e2e tests.

Reply:
In that case we can have a web-test env

Reply:
done